src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -3943,6 +4087,67 @@ export class ResultsViewPageStore {
         },
     });
 
+    readonly getStructuralVariantPutativeDriverInfo = remoteData({
+        await: () => {
+            const toAwait = [];
+            if (this.driverAnnotationSettings.oncoKb) {
+                toAwait.push(
+                    this.getOncoKbStructuralVariantAnnotationForOncoprint
+                );
+            }
+            if (this.driverAnnotationSettings.hotspots) {
+                toAwait.push(this.isHotspotForOncoprint);
```

Comment:
why are you awaiting these (hotspots, cbioportalCount, cosmicCount), they dont' seem to be used?

Reply:
I'll remove them